src/UserService.Mappers/UserMapper.cs

```diff
@@ -1,17 +1,20 @@
-﻿using System;
+﻿using LT.DigitalOffice.UserService.Mappers.Interfaces;
+using LT.DigitalOffice.UserService.Models.Db;
+using LT.DigitalOffice.UserService.Models.Dto;
+using System;
 using System.Linq;
-using LT.DigitalOffice.UserService.Mappers.Interfaces;
 using System.Security.Cryptography;
 using System.Text;
-using LT.DigitalOffice.UserService.Models.Db;
-using LT.DigitalOffice.UserService.Models.Dto;
 
 namespace LT.DigitalOffice.UserService.Mappers
 {
     /// <summary>
     /// Represents mapper. Provides methods for converting an object of <see cref="DbUser"/> type into an object of <see cref="User"/> type according to some rule.
     /// </summary>
-    public class UserMapper : IMapper<DbUser, User>, IMapper<DbUser, string, object>, IMapper<UserCreateRequest, DbUser>, IMapper<EditUserRequest, DbUser>
+    public class UserMapper : IMapper<DbUser, User>, IMapper<DbUser, string, object>,
+        IMapper<UserCreateRequest, DbUser>, IMapper<EditUserRequest, DbUser>,
+        IMapper<EditUserRequest, string, DbUserCredentials>,
+        IMapper<UserCreateRequest, Guid, DbUserCredentials>
```

Comment:
Maybe we should separate User mappers and UserCredentials mappers?

Reply:
Maybe)

Reply:
Fixed